tvtk/vtk_module.py

```diff
@@ -31,3 +31,10 @@
     # Cause problems if used so ignore these.
     SKIP = ['vtkDataEncoder', 'vtkWebApplication']
     del vtkDataEncoder, vtkWebApplication
+
+if vtk_version == '9.1.0':
+    SKIP = ['vtkOpenGLAvatar']
+    try:
+        del vtkOpenGLAvatar
+    except NameError:
```

Comment:
i'm not sure why we're expecting an exception to be raised here.

Reply:
Well, I am not sure if there are build options that may exclude the building of these classes so I was just being extra safe.  The issue may arise if someone has a hand-built VTK.